MONAI bundle app inference + training tests

tangy5 opened the pull request:
An inference integration test scripts, for draft review.

SachidanandAlle commented on tests/integration/monaibundle/test_infer.py:
```diff
@@ -0,0 +1,67 @@
+# from tests1.integration import SERVER_URI
+
+
+SERVER_URI = f"http://0.0.0.0:{os.getenv('MONAILABEL_SERVER_PORT', '8000')}"
```
may be not needed.. you can use `from tests.integration import SERVER_URI`

tangy5 replied:
Yes, modified. Thanks

SachidanandAlle commented:
https://github.com/Project-MONAI/MONAILabel/blob/main/CONTRIBUTING.md#the-contribution-process

and when you do git commit use `-s` option to handle DCO

tangy5 commented:
I modified the inference test, and added a simple trainer test for four bundles in the monaibundle app tests.
Added run_integration_test in ./runtests.sh file.

Seems the trainer test will run training 50 epochs, can we add --conf option to the max_epochs for trainer tests?


Thanks

SachidanandAlle commented:
https://github.com/Project-MONAI/MONAILabel/blob/main/tests/unit/endpoints/test_train.py#L21-L35

Nic-Ma commented:
Hi @tangy5 ,

Glad to see your contribution in MONAI Label.
Please sign off your commits, refer to: https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#signing-your-work.

Thanks.

Nic-Ma commented:
Hi @SachidanandAlle ,

I see the CI failed for:
RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx

Could you please help share some guides here? I imagine @tangy5 needs to modify some CI config.

Thanks in advance.

tangy5 commented:
Hi @SachidanandAlle , one question here,
One check is failed currently saying 
`RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx666`
This is probably due to the none GPU CI test available?, how can we skip this check?
Thank you.

SachidanandAlle commented:
https://github.com/Project-MONAI/MONAILabel/blob/main/tests/integration/radiology/test_infer.py#L22

SachidanandAlle commented:
This might need to change if we are fixing something here: https://github.com/Project-MONAI/model-zoo/issues/79

tangy5 commented:
Thanks！ I will modify if the model_info.json is changed.
